mixer/cmd/repos.go

```diff
@@ -164,9 +171,18 @@ func runSetURLRepo(cmd *cobra.Command, args []string) {
 		fail(err)
 	}
 
-	err = b.SetURLRepo(args[0], args[1])
+	u, err := url.Parse(args[1])
 	if err != nil {
 		fail(err)
 	}
-	fmt.Printf("Set %s baseurl to %s.\n", args[0], args[1])
+	if u.Scheme == "" {
+		u.Scheme = "file"
+	}
+
+	err = b.SetURLRepo(args[0], u.String())
+	if err != nil {
+		fail(err)
+	}
+
+	fmt.Printf("Set %s baseurl to %s.\n", args[0], u.String())
```

Comment:
Why do we need this here?

Reply:
This was already in the code, I just changed the return string of baseurl with the scheme. This is runSetURLRepo(), after updating the url of the repo, it is telling user what the url is updated to.

Reply:
`SetURLRepo()` calls `AddRepo()`. So if you add the logic in `AddRepo()` including the print statement, you can remove this.

Reply:
As you mentioned, since AddRepo() is called only if repo does not exist, this review comment can be ignored.